src/plugin.scss

```diff
@@ -1,23 +1,26 @@
 .video-js {
-  $bottom: 4.5em;
+  $bottom: 40px;
   $nudge: 5px;
   $middle: 50%;
   $offset-h: -16.5%;
   $offset-v: -15px;
 
   .vjs-overlay {
+    color: #fff;
+    position: absolute;
+    text-align: center;
+    max-width: 33%;
+  }
 
+  .vjs-overlay-background {
     // IE8
     background-color: #646464;
     background-color: rgba(255, 255, 255, 0.4);
     border-radius: round($nudge / 2);
-    color: #fff;
     padding: $nudge * 2;
```

Comment:
By putting it here, when the user doesn't opt for the background the overlay will more snugly fit into the side/corner of the player which was one of the requested improvements when using the overlay with an image.